app/models/form_cache.rb

```diff
@@ -31,6 +30,19 @@ def self.fetch_all_analysis(short_uuid)
     end
   end
 
+  def self.fetch_performance_gov_analysis(short_uuid)
```

Comment:
I'd like to be able to force reset this cache, if needed.

When Data Collections come in, they typically come in, like 60 in a week.
I want to be sure we're able to see accurate summary data even when there's lots of activity; entering data that is used in those summary totals.

Reply:
I've added a class method in FormCache to invalidate_reports.   That method will be called on the form#responses path if a parameter called use_cache is present and has a value of false.  Example:
http://lvh.me:3002/admin/forms/bd4b4302/responses?use_cache=false